### ISSUE TITLE
when benchmarking sniffdog is only mining with 9 out of 10 gpu

### ISSUE THREAD

**Issue by chrisdhoo** [Sniffdog/Sniffdogminer#174]
I can use dstm to zec and nicehash with no problem using all 10 gpu. how can I get sniff to detect the 10th?

**Comment by Sniffdog** [comment]
interesting... in the miners folder is all the miners... right click and edit and scroll down to optimizatons..... ccminers use -d 0,1,2,3,4,5,6,7,8,9

**Comment by chrisdhoo** [comment]
I did not see 0,1,2,3,4,5,6,7,8,9 in any of the cc miners under optimizations. I went ahead and let it benchmark everything. I tried to run dstm and nicehash after and both failed to launch. I restarted the computer and ran sniffdog it was mining with all 10 gpu.

i tried to get it to benchmark all 10 several times. each time it started with 10 on the first benchmark then only did 9 on the rest.

**Comment by Sniffdog** [comment]
yeah you need to add in the quotes -d 0,1,2,3,4,5,6,7,8,9 to the algo which you are mining at in optimizations

not sure how you are mining with 10 cards.... unless you are running on same lane multi cards.... just make sure your not dropping your drivers from all the cards you have so what i mean is keep checking your device manager to make sure all 10 cards are recognized

**Comment by chrisdhoo** [comment]
thanks! Ill see if I can find where to add it. i have a problem with the rig crashing after every few hours. Going to try and change my overclock settings. need to see if its crashing on a specific algo.

with the new windows update and asus b250 mining expert i can run up to 13 1080 ti cards. I got one more in the mail and will be running 11 soon.  drivers stay stable and always show working in device manager.

**Comment by chrisdhoo** [comment]
could you show me where to add -d 0,1,2,3,4,5,6,7,8,9

$Path = ".\Bin\NVIDIA-Alexis\ccminer-alexis.exe" 
$Uri = "http://ccminer.org/preview/ccminer-hsr-alexis-x86-cuda8.7z" 


$Name = (Get-Item $script:MyInvocation.MyCommand.Path).BaseName

$Algorithms = [PSCustomObject]@{
    
    #Qubit = 'qubit'
    #NeoScrypt = 'neoscrypt'
    X11 = 'x11'
    MyriadGroestl = 'myr-gr'
    #Groestl = 'groestl'
    #Keccak = 'keccak'
    Scrypt = 'scrypt'
    #Bitcore = 'bitcore'
    Blake2s = 'blake2s'
    Sib = 'sib'
    #X17 = 'x17'
    #Quark = 'quark'
    #Hmq1725 = 'hmq1725'
    Veltor = 'veltor'
    X11evo = 'x11evo'
    #Timetravel = 'timetravel'
    Blakecoin = 'blakecoin'
    #Lbry = 'lbry'
    C11 = 'c11'
    Nist5 = 'nist5'
    Hsr = 'hsr' 
    BlakeVanilla = 'vanilla'
    Lyra2RE2 = 'lyra2v2'
    Skein = 'skein'
    #Skunk = 'skunk'
    
}

**Comment by chrisdhoo** [comment]
where would I add -d 0,1,2,3,4,5,6,7,8,9

![question 0-9](https://user-images.githubusercontent.com/37151634/37241272-35e53f4e-241c-11e8-88b6-def27dc07669.jpg)

**Comment by Sniffdog** [comment]
in the quotations

**Comment by chrisdhoo** [comment]
got all the ccminers updated. not sure how or if to do sgminer, hsr, ewbf, excavator

**Comment by Sniffdog** [comment]
sgminer is not for nvidia cards
ewbf.... -d 0 1 2 3 4 5 6 7 8 9   (no commas)
excavator is setup already for 19 cards
if you use excavator you won't need hsrminer
